source/features/repo-icon-buttons.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import PlusIcon from 'octicon/plus.svg';
+import SearchIcon from 'octicon/search.svg';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+import {groupButtons} from '../github-helpers/group-buttons';
+
+function init(): void {
+	const searchButton = select('.btn[data-hotkey="t"]')!;
+	searchButton.classList.remove('mr-2');
+	searchButton.firstChild!.replaceWith(<SearchIcon/>);
+
+	const addButtonWrapper = searchButton.nextElementSibling!;
+	const addButton = select('summary > span:first-child', addButtonWrapper)!;
```

Comment:
This selector is still not specific enough

Reply:
Yeah this is a bit tough to select. Maybe this is unavoidable:
```suggestion
	const addButton = searchButton.nextSiblingElement!.querySelector('.dropdown-caret')!.parentElement;
```